scripts/test-payload.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+source .venv/bin/activate
+python scripts/generate-linestring.py 5000
+PAYLOAD=$(cat payload.json)
+curl -F "geom=${PAYLOAD}" \
+     -F "elevation_models=COMB" \
+     -H "Referer: https://map.geo.admin.ch" \
```

Comment:
For tests, we should use dedicated referer. That way, we can filter them easily in the logs. something like unittest.geo.admin.ch or test.geo.admin.ch that let's you know the origin of the requests in the logs.

Reply:
This was to test locally. Not on prod.

:)

But in general yes makes sense